### PR TITLE
Add multiple meeting capability to docs generator

### PR DESCRIPTION
**Pull request by cblecker**
I know this will initially fail tests because I'm only regenerating for sig-contribex to start, but this should allow for a sig to have multiple meetings listed.

Feedback welcome!

### CODE REVIEW

**Comment by parispittman** [comment]
Talked to @spiffxp about this yesterday as I was trying to figure out whether to add the second meeting to the comments section of the yaml or if another approach was best. Thanks @cblecker!

File: generator/sig_readme.tmpl [hunk 1]
```diff
@@ -2,18 +2,18 @@
 # {{.Name}} SIG
 
 {{ .MissionStatement }}
-## Meetings
+{{ if .Meetings }}## Meetings
 {{- range .Meetings }}
-* [{{.Day}}s at {{.Time}} {{.TZ}}]({{$.MeetingURL}}) ({{.Frequency}}). [Convert to your timezone](http://www.thetimezoneconverter.com/?t={{.Time}}&tz={{.TZ | tzUrlEncode}}).
+* {{.Description}}: [{{.Day}}s at {{.Time}} {{.TZ}}]({{.URL}}) ({{.Frequency}}). [Convert to your timezone](http://www.thetimezoneconverter.com/?t={{.Time}}&tz={{.TZ | tzUrlEncode}}).
+{{- if .ArchiveURL }}
+  * Meeting notes and Agenda can be found [here]({{.ArchiveURL}}).
```
Comment (jamiehannaford) [inline generator/sig_readme.tmpl]: how do you feel about just changing this to `[Meeting notes and Agenda]({{.ArchiveURL}})`?

File: generator/sig_readme.tmpl [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
 {{- end }}
-
-{{ if .MeetingArchiveURL -}}
-Meeting notes and Agenda can be found [here]({{.MeetingArchiveURL}}).
+{{- if .RecordingsURL }}
+  * Meeting recordings can be found [here]({{.RecordingsURL}}).
```
Comment (jamiehannaford) [inline generator/sig_readme.tmpl]: same with this: `[Meeting notes and Agenda]({{.ArchiveURL}})`

File: generator/sig_readme.tmpl (hunk 1 above)
Comment (cblecker) [reply generator/sig_readme.tmpl]: I like it! *changes*

**Comment by cblecker** [comment]
Updated: https://github.com/cblecker/community/blob/e3d4925acf1b18229831ef1c76f4766068ddd450/sig-contributor-experience/README.md

**Comment by spiffxp** [comment]
/lgtm 

I would like try and redo some of this to move some meetings to subprojects but LGTM so I can rebase off of this.  Thanks for saving me the effort of fleshing out the `Meeting` struct :)

**Pull request merged**